app/services/expired_letters.rb

```diff
@@ -0,0 +1,36 @@
+class ExpiredLetters
+  def call
+    notifications_sent = 0
+    UspsConfirmationCode.where(
+      sql_query_for_expired_letters,
+      tvalue: Time.zone.now - Figaro.env.usps_confirmation_max_days.to_i.days,
+    ).order('created_at ASC').each do |ucc|
```

Comment:
Would a long form name be more readable, e.g. `user_confirmation_code` here an elsewhere?

Reply:
Not following...

Reply:
I'm suggesting changing `ucc` to the slightly longer name `user_confirmation_code` so it's more clear what it represents.

Reply:
ah ok.

Reply:
usps_confirmation_code :-)

Reply:
Ah yes, my b